<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QDockWidget" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="plotWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QFrame" name="bottomFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="horizontalSpacing">
        <number>5</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_Results">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QComboBox" name="comboBox_value">
            <item>
             <property name="text">
              <string>Red (pixel normalized)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red (region normalized)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NbPixel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sd Red (normalized)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sd Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sd Green</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_regions"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_PointPlot">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QComboBox" name="comboBox_point"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_channel">
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red/Green</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox_type">
         <item>
          <property name="text">
           <string>mesh</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>regions</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>external/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
